chain/chain/src/chain.rs

```diff
@@ -4599,6 +4600,10 @@ pub struct BlockCatchUpResponse {
     pub results: Vec<(ShardId, Result<ShardUpdateResult, Error>)>,
 }
 
+#[derive(actix::Message, Debug, Clone, PartialEq, Eq)]
+#[rtype(result = "()")]
+pub struct ReceiveChunkStateWitnessMessage(pub EncodedChunkStateWitness);
```

Comment:
can this simply be ChunkStateWitnessMessage or ProcessChunkStateWitnessMessage? I just find "Receive" confusing. or Accept... maybe?

Reply:
Yeah, I guess ProcessChunkStateWitnessMessage should be good. Will rename! We've already used ChunkStateWitnessMessage for receiving the older type of message, so that'll go away. Perhaps I can rename this to ChunkStateWitnessMessage then.